src/tests/containerizer/cgroups2_tests.cpp

```diff
@@ -82,6 +87,10 @@ class Cgroups2Test : public TemporaryDirectoryTest
   {
     TemporaryDirectoryTest::SetUp();
 
+    // Move ourselves into the root cgroup, in case we moved out of the
+    // root cgroup during a previous test and didn't move back.
+    ASSERT_SOME(cgroups2::assign(cgroups2::ROOT_CGROUP, ::getpid()));
```

Comment:
can we avoid assuming that we are running in the root cgroup? it doesn't seem like something we need to assume, since the mesos code in general doesn't assume this already, it'd be nice if the tests also don't assume this?

Reply:
Removed.